cinema/serializers.py

```
@@ -86,8 +86,12 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], attrs["seat"], attrs["movie_session"]
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"],
+            serializers.ValidationError
```

Issue: The call to `Ticket.validate_ticket` passes `serializers.ValidationError` as an argument, which is likely not intended. Typically, a validation method should raise an exception rather than receive the exception class as an argument. Please check the signature of `Ticket.validate_ticket` in your models and adjust the call accordingly to match its requirements. This is a logic error that may cause validation to not work as expected.